### PR TITLE
Improvements for Laravel tutorial inspired by CUJ

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: tutorials/run-laravel-on-appengine-flexible/index.md [hunk 1]
```diff
@@ -45,16 +45,26 @@ Welcome page.
...
           APP_LOG: errorlog
+          APP_KEY: YOUR_APP_KEY
           STORAGE_DIR: /tmp
 
+1. Replace `YOUR_APP_KEY` in `app.yaml` with an application key you generate
+  with the following command:
+
+        php artisan key:generate --show
+
+    > If you're on Linux or Mac OSX, the following command will automatically
```
Comment (jmdobry) [inline tutorials/run-laravel-on-appengine-flexible/index.md]: Is this section supposed to be a blockquote?

File: tutorials/run-laravel-on-appengine-flexible/index.md [hunk 2]
```diff
@@ -75,6 +85,10 @@ Welcome page.
 
 ## Set up Database Sessions
 
+> Note: This section only works with Laravel 5.4.16. To use earlier versions of
```
Comment (jmdobry) [inline tutorials/run-laravel-on-appengine-flexible/index.md]: I would bold this section instead of using a blockquote.

**Comment by bshaffer** [comment]
Thanks guys!

**Comment by bshaffer** [comment]
Please LMK when this gets published. Thank you!

**Comment by jmdobry** [comment]
Done

**Pull request merged**